### ISSUE TITLE
Ordering for Newly Created Relationships?

### ISSUE THREAD

**Issue by bahamagician** [Hube2/acf-post2post#33]
Thanks for this great plugin.  It's been an answer to my prayers.

I have two post types.  Products and Groups.  When editing a product I use the Post Object field to assign the product to a group.  When I edit the group there's a relationship field where I can see all the products related to the group.  Any products assigned a group from the product edit screen are added to the end of the list of the relationship field within the group edit screen.  Can I change that somehow so that products assigned to a group are added to the beginning of the group relationship list?

**Comment by Hube2** [comment]
Yes, see the description under overwrite settings https://github.com/Hube2/acf-post2post#overwrite-settings
